spec/services/user_creator_spec.rb

```diff
@@ -4,22 +4,35 @@
 
 describe UserCreator, type: :service do
   describe '#call' do
-    let!(:users) { create(:role, name: 'User') }
+    let(:users) { create(:role, name: 'User') }
+    let(:teachers) { create(:role, name: 'Teachers') }
+    let(:fake_setting_getter) { instance_double(SettingGetter) }
 
     before do
       setting = create(:setting, name: 'RoleMapping')
       create(:site_setting, setting:, provider: 'greenlight', value: 'Decepticons=@decepticons.cybertron,Autobots=autobots.cybertron')
     end
 
-    it 'creates a user with the role matching a rule if role found' do
+    it 'creates a user with the default role' do
```

Comment:
Isn't this test duplicated in favor of the last two examples?

Reply:
What last two examples? There is definitely some redundancy in the user_creator_specs. This test isn't really adding anything I agree.